core/ordering/cosipbft/proc.go

```diff
@@ -250,3 +286,20 @@ func (h *processor) makeAccess(store store.Snapshot, roster authority.Authority)
 
 	return nil
 }
+
+// catchupHandler listens to incoming requests for potentially missing blocks.
+// It is started as a go-routine
+func (h *processor) catchupHandler() {
+	for players := range h.catchup {
+		if h.fastsync {
+			ctx, cancel := context.WithCancel(context.Background())
+			err := h.fsync.Sync(ctx, players,
+				fastsync.Config{SplitMessageSize: DefaultFastSyncMessageSize})
+			if err != nil {
+				h.logger.Err(err)
+			}
+			cancel()
+		}
+	}
+	panic("Should not get here")
```

Comment:
please either change this to a more meaningful error message, e.g. including the reason why or remove it

Reply:
It only gets there if the channel is closed, but there is no code to do that. Should I keep the `panic` anyway, but with a message to say that the channel got closed?